src/Impostor.Api/Innersloth/GamePrivacy.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Impostor.Api.Innersloth
+{
+    public enum GamePrivacy
```

Comment:
Can't this be just a bool?

Reply:
yes but i think this makes more sense

Reply:
I also think it should be a bool, since all you do with this enum is
`IsPublic = privace == GamePrivacy.Public;`

Reply:
why don't make it just `Default `, `Private` or `Public`? so, default is configured in settings, like always private or always public?

Reply:
> why don't make it just `Default`, `Private` or `Public`? so, default is configured in settings, like always private or always public?

That doesn't make sense. The point of the method is to set the privacy of the room by plugin. What would `Default` mean ?
Having a setting in `config.json` to set the default privacy could be added, but I don't think it would be useful.